recipes/bids-validator-deno/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "bids-validator-deno" %}
+{% set version = "2.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/bids_validator_deno-{{ version }}.tar.gz
+  sha256: 55a46dc9e134c2996ecb077896aad65e5320f3c864b41c47e108011f8fd1e2d1
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [linux]
```

Comment:
What is the reason this is skipped on Linux? This also shouldn't need the `missing_dso_whitelist` below.

Reply:
I can unskip it for us to check out the error, but here are the previous run's logs: https://github.com/conda-forge/staged-recipes/runs/43674127064

Reply:
Here are the raw logs: https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/1269449/logs/24

The relevant error is this:

```sh
export PREFIX=/home/conda/staged-recipes/build_artifacts/bids-validator-deno_1750081514324/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh
export SRC_DIR=/home/conda/staged-recipes/build_artifacts/bids-validator-deno_1750081514324/test_tmp
+ pip check
No broken requirements found.
+ bids-validator-deno --help
[0m[1m[31merror[0m: Could not find standalone binary section.
```

Reply:
I believe it is a binary executable. 

Tagging @effigies for some more clarity.

Reply:
In that case I would say two things need to checked.  First we should verify the binaries are actually being built from source.  And if they are, then we should try disabling relocation, as there are cases where even a binary built from source gets broken by the relocation.  Most of the time the binary doesn't actually need to be relocated so disabling doesn't create any issues.

Reply:
> I believe it is a binary executable. 
> 
> Tagging @effigies for some more clarity.

Sorry, I haven't had an opportunity to look at this and will be out of contact for the next month.

This package is a typescript project, compilable with deno compile. I don't know how deno packages are handled in conda.

Reply:
I don't have a good understanding of typerscript/deno. I guess we will wait for @effigies to get back so that we can understand where the binaries are being relocated after build.